omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -979,32 +964,6 @@
 		</property>
 	</bean>
 	
-    <bean parent="obsServiceTarget" >
```

Comment:
Do you mind explaining why we removed this?

Reply:
Having this causes an error.


`Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'obsServiceTarget' available`